### PR TITLE
start of a further material appendix, to echo the serveless whitepaper

### PR DESCRIPTION
**Pull request by ramin**
Starting a new section, to follow some of the structure of the serverless-wf whitepaper which @caniszczyk highlighted as successfully executed whitepaper in the meeting before last.

if this is generally in the right direction and approved i'll continue to fill it out with more materials (a mix of case studies, papers, presentations, and writing)

### CODE REVIEW

File: WHITEPAPER.md [hunk 1]
```diff
@@ -195,3 +195,17 @@ The following phases may or may not be useful. I think it would be valuable if w
...
 * [Wiremock](http://wiremock.org/) - API mocking (Service Virtualization) which enables modeling real world faults and delays
+
+
+## Appendix A: Additional Material
+
+- [Principles of Chaos Engineering](http://principlesofchaos.org/)
+
+- [Chaos Engineering: Building Confidence in System Behavior through Experiments](https://www.oreilly.com/webops-perf/free/chaos-engineering.csp)
+
+- [Chaos Engineering: Why Breaking Things Should Be Practised (presentation)](https://www.slideshare.net/hornsby/chaos-engineering-why-breaking-things-should-be-practised-96719638/hornsby/chaos-engineering-why-breaking-things-should-be-practised-96719638)
```
Comment (veggiemonk) [inline WHITEPAPER.md]: It's missing the author: `
Adrian Hornsby, Technical Evangelist at Amazon Web Services`

File: WHITEPAPER.md [hunk 2]
```diff
@@ -195,3 +195,17 @@ The following phases may or may not be useful. I think it would be valuable if w
 * [The Simian Army](https://github.com/Netflix/SimianArmy) - A suite of tools for keeping your cloud operating in top form.
 * [Toxiproxy](https://github.com/Shopify/toxiproxy) - A TCP proxy to simulate network and system conditions for chaos and resiliency testing.
 * [Wiremock](http://wiremock.org/) - API mocking (Service Virtualization) which enables modeling real world faults and delays
+
+
+## Appendix A: Additional Material
+
+- [Principles of Chaos Engineering](http://principlesofchaos.org/)
+
+- [Chaos Engineering: Building Confidence in System Behavior through Experiments](https://www.oreilly.com/webops-perf/free/chaos-engineering.csp)
```
Comment (veggiemonk) [inline WHITEPAPER.md]: It's missing the author and book reference.
By the way, the whole book seems to be available here for free (no signup required) https://www.oreilly.com/ideas/chaos-engineering

**Comment by veggiemonk** [comment]
@ramin ping ?

**Comment by ramin** [comment]
@veggiemonk is there agreement on adding this section and rough direction? if so i'll finish up tonight and build out with a few more references and fuller entries and open for review. Anyone else we need to chime in?

**Comment by veggiemonk** [comment]
@ramin I agree to add an appendix to list further resources but it would be nice to have the approval of another member as well.

**Comment by ramin** [comment]
great i'll finish it up over coming days, thank you!

**Comment by veggiemonk** [comment]
ping @ramin

**Pull request merged**